package.json

```diff
@@ -72,5 +72,8 @@
       "napi-macros/example": true,
       "husky": true
     }
+  },
+  "dependencies": {
+    "dotenv": "^16.3.1"
```

Comment:
https://github.com/ubiquity/ubiquity-dollar is a mono repository which means that if some dependency is added to the top `package.json` file then it is required to be used by all of the [packages](https://github.com/ubiquity/ubiquity-dollar/tree/development/packages)

The `contracts` package already has this [dependency](https://github.com/ubiquity/ubiquity-dollar/blob/development/packages/contracts/package.json#L19)

Either remove the `dotenv` dependency from the top level `package.json` and add it to the `dapp` package either remove the `dotenv` dependency from the `contracts` package to keep a single one in the top level `package.json`